modules/attribution-document/attribution-document-core/src/main/java/org/eclipse/sw360/antenna/attribution/document/utils/TemplateLoaderUtil.java

```diff
@@ -78,9 +86,66 @@ public InputStream loadBackPageTemplate() {
                     throw new ExecutionException("Could not load back template PDF", e);
                 }
             }
+
+            @SuppressFBWarnings(value = "OBL_UNSATISFIED_OBLIGATION", justification = "Will be closed in Templates.")
+            @Override
+            public Optional<InputStream> loadSansFont() {
+                if (regular.isPresent()) {
```

Comment:
Just a hint about what we're doing in ORT, maybe you like it: We write `if` / `else` statements like this to return early on the negated statement, like

    if (!regular.isPresent()) return Optional.empty();

    // Rest of code from original `if`-branch.

This was the advantage to reduce nesting / indentation for a better overview.